### ISSUE TITLE
Why are there both new_scenario and ocean_scenario classes?

### ISSUE THREAD

**Issue by denised** [ProjectDrawdown/solutions#520]
Seems like we should need just one of these...
@glass-wing, it looks like ocean_scenario is the code in use.  Does any of your code actually use new_scenario.py?

**Comment by glass-wing** [comment]
Hi Denise.

Happy new year!
I'm going to pick this up again on Monday. Haven't looked at it in over a
month now. Lots of cobwebs.
Hope you're well.

Rgds
Neil

**Comment by glass-wing** [comment]
You're right - ocean_scenario is the only one I use. I did a search across
all files in my fork, and found no matches.

Rgds
Neil

**Comment by denised** [comment]
OK then, I will remove it --- thanks for checking.
